packages/block-library/src/query-loop/edit.js

```diff
@@ -41,6 +41,7 @@ export default function QueryLoopEdit( {
 			search,
 			exclude,
 			sticky,
+			isGlobalQuery,
```

Comment:
What do you think of calling this attribute `inherit` instead?

Reply:
Hmmm since it's part of the `query`, inherit makes sense. Done in https://github.com/WordPress/gutenberg/pull/27128/commits/ccf3513e4af3ebe3154fcc061a09b8fc9b6f5601 :+1:

Reply:
Ideally, we shouldn't need to pass it down but rather change the context passed down similar to how it happens in React with a provider. I plan to look into that this week once this PR is merged.